src/main/java/org/jabref/gui/preftabs/AppearancePrefsTab.java

```diff
@@ -60,8 +60,6 @@
         public static Set<String> getAvailableLookAndFeels() {
 
             Set<String> lookAndFeels = new HashSet<>();
-            lookAndFeels.add("com.jgoodies.looks.plastic.Plastic3DLookAndFeel");
-            lookAndFeels.add("com.jgoodies.looks.windows.WindowsLookAndFeel");
 
             for (LookAndFeelInfo info : UIManager.getInstalledLookAndFeels()) {
```

Comment:
I think the rest of the method can now be replaced by the one-liner `return getInstalledLookAndFeels().stream().map(LookAndFeelInfo:getClassName).collect(toSet)`  ?

Reply:
Not exactly (it's and array), but almost: `Arrays.stream(UIManager.getInstalledLookAndFeels()).map(LookAndFeelInfo::getClassName).collect(Collectors.toSet())`